docs/installing.adoc

```diff
@@ -293,27 +271,37 @@ $ vagrant box add adb <local path to the downloded image>
 [[set-up-adb-behind-an-http-proxy]]
 == Step 6. (Optional) Set up ADB behind an HTTP proxy
 
-ADB can be set up behind a proxy server. You need to export the proxy
-server information in to the environment and then run `vagrant up`.
-
-NOTE: Currently, only HTTP and HTTPS proxy servers are supported.
-
-For Linux, OS X and Windows Cygwin shell:
+ADB can be set up behind a proxy server. Set the following proxy parameters in the Vagrantfile you chose in the previous step:
 
 ....
-export PROXY="<proxy_server>:<port>"
-export PROXY_USER="foo"
-export PROXY_PASSWORD="mysecretpass"
+config.servicemanager.http_proxy = <Proxy URL>
+config.servicemanager.http_proxy_user = <Proxy user name>
+config.servicemanager.http_proxy_password = <Proxy user password>
 ....
 
-For Windows CMD or Powershell:
+Once these settings are applied, they are passed through to the Docker and OpenShift services to enable them to function correctly.
+In an unauthenticated proxy environment, the `http_proxy_user` and `http_proxy_password` configurations can be omitted. 
```

Comment:
@Preeticp I am not able to understand purpose of `In an unauthenticated proxy environment, the `http_proxy_user` and `http_proxy_password` configurations can be omitted. ` Can you please add more context.